modules/tinymce/src/themes/silver/test/ts/browser/editor/contextform/ContextFormTest.ts

```diff
@@ -409,12 +449,34 @@ describe('browser.tinymce.themes.silver.editor.ContextFormTest', () => {
     UiFinder.exists(SugarBody.body(), '.tox-pop input[placeholder="This is a placeholder"]');
   });
 
-  it('TINY-11559: Focus should be on toolbar when onSetup disables the main input', () => {
+  // TODO: fix and re-enable it or replace/remove it
```

Comment:
Rewrite this to check that the button gets the focus if clicked and that button invokes the setInputEnabled to false.

Reply:
I tested the: `invokes the setInputEnabled to false.` but checking the focus on the button seems not possible since as soon as the button is clicked it disappear to make space for the input.

Maybe I'm missing something, but if you think the new version is ok resolve this conversation

Reply:
The toolbar should be the a button that disables the input like we have in the alt text. Then we check if the focus is moved to the button that clicks the setInputEnabled call.

Reply:
done